libqfieldsync/utils/qgis.py

```diff
@@ -48,12 +48,22 @@ def open_project(filename: str, filename_to_read: Optional[str] = None) -> bool:
     return is_success
 
 
-def make_temp_qgis_file(project: QgsProject) -> str:
+def make_temp_qgis_file(
+    project: QgsProject,
+    exported_filename: Optional[str] = None,
+    exported_project_title: Optional[str] = None,
+) -> str:
     project_backup_dir = tempfile.mkdtemp()
     original_filename = project.fileName()
-    backup_filename = os.path.join(project_backup_dir, f"{project.baseName()}.qgs")
-    project.write(backup_filename)
-    project.setFileName(original_filename)
+    if exported_filename and exported_project_title:
+        backup_filename = os.path.join(project_backup_dir, f"{exported_filename}.qgs")
+        project.write(backup_filename)
+        project.setTitle(exported_project_title)
+        project.setFileName(backup_filename)
+    else:
+        backup_filename = os.path.join(project_backup_dir, f"{project.baseName()}.qgs")
+        project.write(backup_filename)
+        project.setFileName(original_filename)
```

Comment:
```suggestion
    project.write(exported_filename)
    project.setFileName(exported_filename)

    if exported_project_title:
        project.setTitle(exported_project_title)
```

I guess something like this.